src/keyring/ed25519/public_key.rs

```diff
@@ -1,38 +0,0 @@
-use iq_bech32::Bech32;
-use signatory::ed25519;
-pub use signatory::ed25519::PUBLIC_KEY_SIZE;
-use std::fmt::{self, Display};
-#[cfg(feature = "yubihsm")]
-use yubihsm;
-
-use error::{KmsError, KmsErrorKind::*};
-
-/// Ed25519 public keys
-#[derive(Copy, Clone, Eq, PartialEq, PartialOrd, Ord)]
-pub struct PublicKey(ed25519::PublicKey);
-
-impl PublicKey {
-    /// Convert a bytestring to a public key
-    pub fn from_bytes(bytes: &[u8]) -> Result<Self, KmsError> {
-        Ok(PublicKey(
-            ed25519::PublicKey::from_bytes(bytes).map_err(|e| err!(InvalidKey, "{}", e))?,
-        ))
-    }
-
-    /// Obtain public key as a byte array reference
-    #[inline]
-    pub fn as_bytes(&self) -> &[u8; PUBLIC_KEY_SIZE] {
-        self.0.as_bytes()
-    }
-}
-
-impl Display for PublicKey {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let bech: Bech32 = Default::default();
-
-        let bech_str = bech.encode("rawed25519", self.as_bytes());
```

Comment:
Are `rawed25519` addresses actually used for anything or are they just a placeholder?

If they're actually used, this code should get moved into the `tendermint` crate